src/vivarium_census_prl_synth_pop/components/household.py

```diff
@@ -32,12 +38,17 @@ def name(self):
     #################
 
     def setup(self, builder: Builder):
+        self.start_time = get_time_stamp(builder.configuration.time.start)
+        self.randomness = builder.randomness.get_stream(self.name)
         self.columns_used = [
             "tracked",
             "household_id",
         ]
 
         self.population_view = builder.population.get_view(self.columns_used)
+        self.state_puma_options = builder.data.load("population.households")[
```

Comment:
We have a csv that contains all possible states and pumas. You can find the csv here: https://vivarium-research.readthedocs.io/en/latest/models/concept_models/vivarium_census_synthdata/concept_model.html#id27. Matt is already using it in ZIP code generation in post-processing, so sync up with him regarding the location where he's putting the file.